config/serverless.yml

```diff
@@ -40,5 +40,6 @@ server.securityResponseHeaders.disableEmbedding: true
 # Enforce single "default" space
 xpack.spaces.maxSpaces: 1
 
-# Allow unauthenticated access to task manager utilization API for auto-scaling
+# Temporarily allow unauthenticated access to task manager & status/stats APIs for autoscaling
+status.allowAnonymous: true
```

Comment:
Unrelated, but not having owners on this file somewhat scares me. Mind adding core as owners, given apparently we're also owners of the main config file

https://github.com/elastic/kibana/blob/9492c932bda777515243f0e19239119349a62838/.github/CODEOWNERS#L935

Reply:
good idea, done.

Reply:
I actually added core to all of `/config/` which also includes the default `node.options` and other serverless config files... if we want to exclude those I can go back and only add the main `serverless.yml`